wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGeneratorTest.java

```diff
@@ -73,13 +83,13 @@ public void setUp() throws Exception {
   public void genOfRunningWorkspaceLinks() throws Exception {
     when(workspace.getStatus()).thenReturn(WorkspaceStatus.RUNNING);
 
-    assertEquals(linksGenerator.genLinks(workspace), expectedRunningLinks);
+    assertEquals(linksGenerator.genLinks(workspace, serviceContextMock), expectedRunningLinks);
   }
 
   @Test
   public void genOfStoppedWorkspaceLinks() throws Exception {
     when(workspace.getStatus()).thenReturn(WorkspaceStatus.STOPPED);
 
-    assertEquals(linksGenerator.genLinks(workspace), expectedStoppedLinks);
+    assertEquals(linksGenerator.genLinks(workspace, serviceContextMock), expectedStoppedLinks);
```

Comment:
Can you add tests with different port and SSL presence?

Reply:
done